src/components/input/input-date-picker.tsx

```diff
@@ -19,15 +17,11 @@ const InputDatePicker: React.FC<InputDatePickerProps> = ({
   return (
     <Label>
       {label}
-      {selected ? (
-        <DatePicker
-          selected={new Date(selected)}
-          onChange={onChange}
-          dateFormat={"yyyy-MM-dd"}
-        />
-      ) : (
-        <SkeletonTextLine fontSize={1.4} />
-      )}
+      <DatePicker
+        selected={selected ? new Date(selected) : undefined}
```

Comment:
What is the default when the date is undefined?

If it's not already "today", maybe we could:
```suggestion
        selected={selected ? new Date(selected) : new Date()}
```

Reply:
Well, at first I tried to go with `Date.now()`, which is an invalid type. I forgot that `new Date()` was initializing the constructor with a current date. I'll make the change.